query/src/sql/sql_parser.rs

```diff
@@ -653,6 +706,7 @@ impl<'a> DfParser<'a> {
     }
 
     fn parse_truncate(&mut self) -> Result<DfStatement, ParserError> {
+        self.parser.next_token();
         match self.parser.next_token() {
```

Comment:
So many `match self.parser.next_token()`

Let's refactor this into `self.parser.expect_token(token)?`

Reply:
It has been verified `parse_statement match`,  may not be necessary to use `expect_token` ？